its/projects/DuplicateAnalyzerReferences/DuplicateAnalyzerReferences.sln

```diff
@@ -0,0 +1,48 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30114.105
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DuplicateAnalyzer", "DuplicateAnalyzer\DuplicateAnalyzer.csproj", "{76169A6D-2D7B-40A4-988F-6A4E3C6CB460}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SourceGenerator", "SourceGenerator\SourceGenerator.csproj", "{B7DDE3C5-5051-44CA-BC02-46C550E4DA3F}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
```

Comment:
I think that if we drop the x86 we'll save some space in overall slns and reduce overall sizes... just sharing a related thought